cmd/openshift-install/create.go

```diff
@@ -435,27 +440,44 @@ func logComplete(directory, consoleURL string) error {
 	if err != nil {
 		return err
 	}
-	logrus.Info("Install complete!")
+	if complete {
+		logrus.Info("Install complete!")
+	}
 	logrus.Infof("Run 'export KUBECONFIG=%s' to manage the cluster with 'oc', the OpenShift CLI.", kubeconfig)
 	logrus.Infof("The cluster is ready when 'oc login -u kubeadmin -p %s' succeeds (wait a few minutes).", pw)
-	logrus.Infof("Access the OpenShift web-console here: %s", consoleURL)
+	if consoleURL != "" {
+		logrus.Infof("Access the OpenShift web-console here: %s", consoleURL)
+	}
 	logrus.Infof("Login to the console with user: kubeadmin, password: %s", pw)
 	return nil
 }
 
 func finish(ctx context.Context, config *rest.Config, directory string) error {
-	if err := waitForInitializedCluster(ctx, config); err != nil {
-		return err
+	err := waitForInitializedCluster(ctx, config)
+	consoleURL, err2 := waitForConsole(ctx, config, rootOpts.dir, err != nil)
```

Comment:
why don't you aggregate all the errors using something like https://godoc.org/k8s.io/apimachinery/pkg/util/errors#Aggregate

Reply:
> why don't you aggregate all the errors...

Done with 6c92c2109 -> d3214acda.